.github/workflows/api-pull-request.yml

```diff
@@ -12,14 +12,16 @@ on:
       - .github/**
     branches:
       - main
+  merge_group:
+    types: [checks_requested]
```

Comment:
What does this do?

Reply:
Runs the jobs of this workflow for merge-queue events, please refer to https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue and https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#merge_group for further details.